### DISCUSSION TITLE
Module not found error

### DISCUSSION THREAD

**Discussion by donovanking** [vercel/community#2815] [Help] [answered]
### Summary

Good evening, 

I am trying to upload my website on Vercel, but I keep getting these errors and I do not know why. It works on my local host perfectly. My website is on my Git Repo if anyone could please take a look at it and let me know what the issue is that would be greatly appreciated!
<img width="607" alt="Screen Shot 2023-06-10 at 7 54 22 PM" src="https://github.com/vercel/community/assets/123105414/86a35448-4939-447d-aef0-a2568c819203">

### Steps to Reproduce

You could just try to run it on Vercel

**Comment by Itz-MrJ** [comment, marked as answer]
Hey @donovanking 
It looks like the modules you are trying to import are not relative. Change it to a relative path and try. That should solve your issue.

Change the import lines from -
`import { section } from "/Users/donovanking/documents/something/section.js"`
To -
`"import {section } from "../../components/section.js"`


The above change is just an example. The change completely goes on your file-folder structure. Make the import line to a relative path and let me know if that works out.

**Reply by donovanking** [reply]
Hey @Itz-MrJ 
I tried that, and I noticed that at the bottom it said "Error: Command 'npm run build' exited with 1." So, I tried fixing the errors on my local because when I did that command I got the same errors. However, even fixing the import lines I still got the exact same error.

**Reply by donovanking** [reply]
Nevermind, got it. Thank you for your time and help! Much appreciated.
